orangecontrib/bioinformatics/resolwe/utils.py

```diff
@@ -58,6 +58,12 @@ def etc_to_table(etc_json, transpose=False):
             remove_redundant_inst=True,
         )
 
+        orange_table.domain = Domain(
+            orange_table.domain.attributes,
+            orange_table.domain.class_vars,
+            [ContinuousVariable('Time')],
+        )
+
```

Comment:
```suggestion
        old_domain = orange_table.domain
        new_domain = Domain(old_domain.attributes,
                            old_domain.class_vars,
                            [ContinuousVariable(var.name) for var in old_domain.metas])

        time_as_floats = orange_table.get_column('Time').astype(int).reshape(-1, 1)
        orange_table = orange_table.transform(new_domain)
        orange_table[:, new_domain['Time']] = time_as_floats
```

Reply:
To avoid some unexpected behaviours it is safe to use Table.transform when we you are changing domain.